src/App.js

```diff
@@ -44,14 +39,14 @@ class App extends Component {
     this.setState({currentColor: color});
   }
   render () {
-
+    const { clickedColorsArray, currentColor, urlValue, imgUrl} = this.state;
     return (
       <div className="App">
 
         <div className="main" >
-        <Header clickedColorsArray={this.state.clickedColorsArray} currentColor={this.state.currentColor}/>
+        <Header clickedColorsArray={clickedColorsArray} currentColor={currentColor}/>
         {/* <div className="bigSwatch" style={{backgroundColor: this.state.currentColor}}> ColorSwatch <div style={{fontSize: '10px'}}>Click anywhere on the picture to get started</div></div> */}
-        <Canvas onClick={this.clickImageHandler} updateImage={this.updateImage} handleChange={this.handleChange} imgUrl={this.state.imgUrl} urlValue={this.state.urlValue} clickedColorsArray={this.state.clickedColorsArray} deleteItem={this.deleteItem} bigColor={this.bigColor}/> 
+        <Canvas onclick={this.clickImageHandler} updateImage={this.updateImage} handleChange={this.handleChange} imgUrl={imgUrl} urlValue={urlValue} clickedColorsArray={clickedColorsArray} deleteItem={this.deleteItem} bigColor={this.bigColor}/> 
```

Comment:
components with a lot of props are usually written with one prop per line, to make it more readable.